imapclient/imapclient.py

```diff
@@ -28,6 +29,9 @@
     long = int  # long is just int in python3
 
 
+logger = getLogger(__name__)
+imaplib_logger = getLogger('imapclient.imaplib')
```

Comment:
I didn't realise that "imapclient" would be doubled up. The way you have it is better.